integration-tests/geocoder/src/main/java/org/apache/camel/quarkus/component/geocoder/it/MockApiService.java

```diff
@@ -21,8 +21,10 @@
 import javax.enterprise.context.ApplicationScoped;
 
 import com.google.maps.GeoApiContext;
+import io.quarkus.runtime.annotations.RegisterForReflection;
 
 @ApplicationScoped
+@RegisterForReflection(targets = GeoApiContext.Builder.class)
```

Comment:
It's actually really cleaner written that way. So, we can remove quarkus.camel.native.reflection at all ?

Reply:
Unlike `@RegisterForReflection`, `quarkus.camel.native.reflection` supports wildcards so they are not fully interchangeable.